frontend/server/src/main/java/org/pytorch/serve/metrics/configuration/MetricTypes.java

```diff
@@ -0,0 +1,53 @@
+package org.pytorch.serve.metrics.configuration;
+
+import java.util.List;
+
+public class MetricTypes {
+    private List<MetricSpecification> counter;
+    private List<MetricSpecification> gauge;
+    private List<MetricSpecification> histogram;
+
+    public void setCounter(List<MetricSpecification> counter) {
+        this.counter = counter;
+    }
+
+    public List<MetricSpecification> getCounter() {
+        return this.counter;
+    }
+
+    public void setGauge(List<MetricSpecification> gauge) {
+        this.gauge = gauge;
+    }
+
+    public List<MetricSpecification> getGauge() {
+        return this.gauge;
+    }
+
+    public void setHistogram(List<MetricSpecification> histogram) {
+        this.histogram = histogram;
+    }
+
+    public List<MetricSpecification> getHistogram() {
+        return this.histogram;
+    }
+
+    public void validate() {
+        if (this.counter != null) {
+            for (MetricSpecification spec : this.counter) {
+                spec.validate();
+            }
```

Comment:
shouldn't return at here?

Reply:
Here, we're validating all the metric configurations that have been loaded, `this.counter` is a list of metric specifications. If there are no counter metrics defined, we skip it and check for the next metric type. So we won't return here after validating only `this.counter`.